packages/react-aria-components/docs/Autocomplete.mdx

```diff
@@ -116,6 +124,452 @@ import {UNSTABLE_Autocomplete as Autocomplete, SearchField, Menu} from 'react-ar
+* **Styleable** – Items include builtin states for styling, such as hover, press, focus, selected, and disabled.
+
+### Concepts
+
+`Autocomplete` makes use of the following concepts:
+
+<section className={styles.cardGroup} data-size="small">
+
+<ExampleCard
+  url="collections.html"
+  title="Collections"
+  description="Defining collections of items, async loading, and updating items over time.">
+  <Collections />
+</ExampleCard>
+
+<ExampleCard
+  url="selection.html"
+  title="Selection"
+  description="Interactions and data structures to represent selection.">
+  <Selection />
+</ExampleCard>
+
+</section>
+
+### Composed components
+
+An `Autocomplete` can use the following components, which may also be used standalone or reused in other components.
+
+<section className={styles.cardGroup} data-size="small">
+
+<ExampleCard
+  url="TextField.html"
+  title="TextField"
+  description="A text field allows a user to enter a plain text value with a keyboard.">
+  <TextField />
+</ExampleCard>
+
+<ExampleCard
+  url="SearchField.html"
+  title="SearchField"
+  description="A search field allows a user to enter and clear a search query.">
+  <SearchField />
+</ExampleCard>
+
+<ExampleCard
+  url="Menu.html"
+  title="Menu"
+  description="A menu displays a list of actions or options that a user can choose.">
+  <Menu />
+</ExampleCard>
+
+<ExampleCard
+  url="ListBox.html"
+  title="ListBox"
+  description="A listbox allows a user to select one or more options from a list.">
+  <ListBox />
+</ExampleCard>
+
+</section>
+
+## Examples
+
+<ExampleList tag="combobox" />
+
+## Starter kits
+
+To help kick-start your project, we offer starter kits that include example implementations of all React Aria components with various styling solutions. All components are fully styled, including support for dark mode, high contrast mode, and all UI states. Each starter comes with a pre-configured [Storybook](https://storybook.js.org/) that you can experiment with, or use as a starting point for your own component library.
+
+<StarterKits component="autocomplete" />
+
+## Reusable wrappers
+
+If you will use an Autocomplete in multiple places in your app, you can wrap all of the pieces into a reusable component. This way, the DOM structure, styling code, and other logic are defined in a single place and reused everywhere to ensure consistency.
+
+This example wraps `Autocomplete` and all of its children together into a single component which accepts a `label` prop and `children`, which are passed through to the right places. It also shows how to use the `description` slot to render [help text](#description), and `FieldError` component to render [validation errors](#validation). The `Item` component is also wrapped to apply class names based on the current state, as described in the [styling](#styling) section.
```

Comment:
the field error/description stuff isn't really shown in this example. maybe omit that since it's covered in search field.